services/app-api/src/domain-models/healthPlanPackage.ts

```diff
@@ -60,9 +69,144 @@ function packageSubmitters(pkg: HealthPlanPackageType): string[] {
     return pruneDuplicateEmails(submitters)
 }
 
+function convertContractToUnlockedHealthPlanPackage(
+    contract: ContractType
+): HealthPlanPackageType | Error {
+    console.info('Attempting to convert contract to health plan package')
+
+    try {
+        const healthPlanRevisions =
+            convertContractRevisionToHealthPlanRevision(contract)
+        return {
+            id: contract.id,
+            stateCode: contract.stateCode,
+            revisions: healthPlanRevisions,
+        }
+    } catch (err) {
+        console.warn(
+            `Error: convertContractToUnlockedHealthPlanPackage encountered an error: ${err.message}`
+        )
+        return err
+    }
+}
+
+function convertContractRevisionToHealthPlanRevision(
+    contract: ContractType
+): HealthPlanRevisionType[] {
+    if (contract.status !== 'DRAFT') {
+        throw new Error(
```

Comment:
please return errors rather than throwing them, is there something else in this function that can throw? I always prefer us to box up dependencies that throw